### ISSUE TITLE
Using vite asset in config file

### ISSUE THREAD

**Issue by lucraraujo** [fruitcake/laravel-weasyprint#2]
I know that this doubt should be posted at laravel forums, but I though that it would be a problem that developers who uses this package can face.

When I try to load my CSS asset from Vite in the weasyprint.php config file I get an error.

I'm trying to do something like this:

```php
<?php


use Illuminate\Support\Facades\Vite;

'pdf' => [
        'options' => [
            'stylesheet' => [
                Vite::asset('resources/sass/app.scss')
            ],
        ],
];
```

But I'm getting this error:

![image](https://github.com/fruitcake/laravel-weasyprint/assets/7658062/c0d22a9b-891f-4b3b-939b-c2501b0cb0e5)

Any ideas on how to inject Vite asset in weasyprint?

Thanks!

**Comment by lucraraujo** [comment]
I manage to workaraound this issue with the example below:
```php
$pdf = WeasyPrint::loadView('pdftemplate');
$pdf->setOption('stylesheet', [Vite::content('resources/sass/app.scss')]);
```

The downside is that the assets must be compiled since I could not make it work with URLs.

**Comment by barryvdh** [comment]
Yeah not sure, this is a limitation of Laravel. But you can just add the stylesheet in your HTML like you would for normale pages.
